README.md

```diff
@@ -110,6 +110,33 @@ It is also possible to start the script with a config file instead of using the
 
 > __Note:__ This should only be used by experienced users as invalid values will cause the framework to fail!
 
+
+#### Using your own modules
+You can specify the applications, mappers, solvers and devices that the benchmark manager should work with by
+specifying a module configuration file with the option '-m|--modules'. The module configuration file has to
```

Comment:
Maybe we can add one sentence here about the advantage of this like: "The advantage of only including specific modules is that you don't need to install the packages used across all modules but only those used by the specified modules."